library/Vanilla/Models/ThemeModelHelper.php

```diff
@@ -41,6 +43,68 @@ public function __construct(
         $this->config  = $config;
     }
 
+    /**
+     * Filter themes based on their addon.json.
+     *
+     * @param AddonInterface $theme A themes data from it's addon.json.
+     * @param string $siteName The vanilla domain of the site.
+     *
+     * @return bool
+     */
+    public function isThemeVisible(AddonInterface $theme, ?string $siteName = null): bool {
+        if ($siteName === null && defined('CLIENT_NAME')) {
+            $siteName = CLIENT_NAME;
+        }
+        $confVisible = $this->config->get('Garden.Themes.Visible', '');
+
+        if ($confVisible === 'all') {
+            // Config setup to show all themes.
+            return true;
+        }
+
+        if ($this->session->getPermissions()->isAdmin()) {
```

Comment:
This will show too many themes. The user that starts an account is admin = 1.

Reply:
Ah ok. I'll check for 2 specifically then, which is our system user.